pci/src/vfio.rs

```diff
@@ -436,7 +436,10 @@ impl VfioCommon {
                         .ok_or(PciDeviceError::IoAllocationFailed(region_size))?;
                 }
                 #[cfg(target_arch = "aarch64")]
-                unimplemented!()
+                {
```

Comment:
Nothing wrong with this. Could we also solve this by moving:

```
// Mask flag bits (lowest 2 for I/O bars)
                    lower &= !0b11;
                    // BAR is not enabled
                    if lower == 0 {
                        bar_id += 1;
                        continue;
                    }
```

Out of the arch conditional code i.e. to just after `if io_bar {` and then that would simplify the conditionals?

Reply:
Done